source/_integrations/flume.markdown

```diff
@@ -31,14 +31,48 @@ To add `Flume` to your installation, go to **Settings** -> **Devices & Services*
 
 ## Notifications
 
-Flume notifications are available via binary sensors. To clear the notifications, you will need to use your Flume app or go to: [https://portal.flumewater.com/notifications](https://portal.flumewater.com/notifications) and clear the notification in question.
-
-The following notifications are supported:
+Flume notifications are fetched every 5 minutes and are available via the service `flume.list_notifications`. Some notifications are available via the following binary sensors:
 
 - Bridge disconnected
 - High flow
 - Leak detected
+- Low battery
+
+To clear the notifications, you will need to use your Flume app or go to: [https://portal.flumewater.com/notifications](https://portal.flumewater.com/notifications) and clear the notification in question.
+
+Example of an automation that sends a Home Assistant notification of the most recent usage alert:
 
+```yaml
+alias: "Notify: flume"
+trigger:
+  - platform: time_pattern
+    minutes: /5
+condition: []
+action:
+  - service: flume.list_notifications
+    data:
+      config_entry: 1234 # replace this with your config entry id
+    response_variable: notifications
+  - if:
+      - condition: template
+        value_template: >-
+          {{ notifications.notifications | selectattr("type", "equalto", 1) |
+          sort(attribute="created_datetime", reverse=true) | length  > 0 }}
```

Comment:
It complains about the unexpected = here. Please advice how to escape it. This yaml works as is in Home Assistant.

Reply:
Thanks for fixing

Reply:
haha, sorry you had to witness my trial-and-error approach on this. I didn't approve yet because I would like to have a second opinion.